hw/ip/otbn/dv/smoke/run_smoke.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+# Runs the OTBN smoke test (builds software, build simulation, runs simulation
+# and checks expected output)
+
+fail() {
+    echo >&2 "$@"
+    exit 1
+}
+
+set -o pipefail
+
+SCRIPT_DIR="$(dirname "$(readlink -e "$BASH_SOURCE")")"
+UTIL_DIR="$(readlink -e "$SCRIPT_DIR/../../../../../util")" || {
+  echo >&2 "Can't find OpenTitan util dir"
+  exit 1
+}
```

Comment:
I should make this use fail

Reply:
Sorry - mid-air collision! Yes, I agree :-)